joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/ImportTests.scala

```diff
@@ -178,4 +178,56 @@ class ImportTests extends RubyCode2CpgFixture with Inspectors {
       }
     }
   }
+
+  "`require_all` on a directory" should {
```

Comment:
Will resolution for `require_relative` be implemented at a later stage?

Reply:
So while I interpret `require_all` and `require_relative` as both starting paths from root, (where only `require_all` is explicitly tested), I should actually also keep an eye out for `$LOADER << ` calls which mean that I can't only consider root alone. Let me add handling for that